### PR TITLE
[hi] Update /data/i18n/hi/hi.toml with new strings from en.toml

### PR DESCRIPTION
**Pull request by rolfedh**
Fixes https://github.com/kubernetes/website/issues/36203

Needs language review from reviewer who is fluent in Hindi.

### CODE REVIEW

**Comment by sftim** [comment]
Thanks for the effort!

File: data/i18n/hi/hi.toml [hunk 1]
```diff
@@ -70,6 +73,9 @@ other = "क्या आप यह ढूँढ रहे थें:"
 [examples_heading]
 other = "उदाहरण"
 
+[feature_state]
+other = "फ़ीचर राज्य:"
```
Comment (anubha-v-ardhan) [inline data/i18n/hi/hi.toml:77]:
```suggestion
other = "फ़ीचर स्थिति:"
```
The word राज्य translates to "state" in geography. (Example: "There are 50 states in the US"). स्थिति is the correct translation here.

**Review by anubha-v-ardhan — CHANGES_REQUESTED** [review]
Thank you @rolfedh :)

File: data/i18n/hi/hi.toml [hunk 2]
```diff
@@ -88,6 +94,9 @@ other = ","
 [input_placeholder_email_address]
 other = "ईमेल ऐड्रेस"
 
+[javascript_required]
+other = "इस सामग्री को देखने के लिए जावास्क्रिप्ट [सक्षम](https://www.enable-javascript.com/) होना चाहिए"
```
Comment (AmanSarraf) [inline data/i18n/hi/hi.toml:98]:
`````````suggestion
other = "इस सामग्री को देखने के लिए जावास्क्रिप्ट [सक्रिय](https://www.enable-javascript.com/) होना चाहिए"
```````````
This looks good here

File: data/i18n/hi/hi.toml (hunk 1 above)
Comment (rolfedh) [reply data/i18n/hi/hi.toml:77]: Updated. Thanks.

File: data/i18n/hi/hi.toml (hunk 2 above)
Comment (rolfedh) [reply data/i18n/hi/hi.toml:98]: Updated. Thanks.

**Comment by rolfedh** [comment]
Thank you both for your comments, @anubha-v-ardhan @AmanSarraf. I've updated the contents and I believe this PR is ready to be reviewed for merge.

File: data/i18n/hi/hi.toml [hunk 3]
```diff
@@ -88,6 +94,9 @@ other = ","
 [input_placeholder_email_address]
 other = "ईमेल ऐड्रेस"
 
+[javascript_required]
+other = "इस सामग्री को देखने के लिए जावास्क्रिप्ट [सक्रिय](https://www.enable-javascript.com/) होना चाहिए"
```
Comment (Babapool) [inline data/i18n/hi/hi.toml:98]:
```suggestion
other = "इस विषय को देखने के लिए जावास्क्रिप्ट [सक्रिय](https://www.enable-javascript.com/) होना चाहिए"
```

File: data/i18n/hi/hi.toml [hunk 4]
```diff
@@ -37,6 +37,9 @@ other = " प्रलेखन अब सक्रिय रूप से म
 [deprecation_file_warning]
 other = "डेप्रिकॉटेड"
 
+[dockershim_message]
+other = """1.24 रिलीज के बाद से डॉकरशिम को कुबेरनेट्स परियोजना से हटा दिया गया है। अधिक जानकारी के लिए <a href="/dockershim">डॉकरशिम रिमूवल एफएक्यू</a> पढ़ें।"""
```
Comment (Babapool) [inline data/i18n/hi/hi.toml:41]:
```suggestion
other = """1.24 संस्करण के बाद से डॉकरशिम को कुबेरनेट्स परियोजना से हटा दिया गया हैं। अधिक जानकारी के लिए <a href="/dockershim">डॉकरशिम रिमूवल एफएक्यू</a> पढ़ें।"""
```

File: data/i18n/hi/hi.toml [hunk 5]
```diff
@@ -37,6 +37,9 @@ other = " प्रलेखन अब सक्रिय रूप से म
 [deprecation_file_warning]
 other = "डेप्रिकॉटेड"
```
Comment (Garima-Negi) [inline data/i18n/hi/hi.toml:38]: can be replaced by `डेप्रिकेटड`

File: data/i18n/hi/hi.toml (hunk 3 above)
Comment (rolfedh) [reply data/i18n/hi/hi.toml:98]: Updated. Thanks.

File: data/i18n/hi/hi.toml (hunk 5 above)
Comment (rolfedh) [reply data/i18n/hi/hi.toml:38]: Updated. Thanks.

File: data/i18n/hi/hi.toml (hunk 4 above)
Comment (rolfedh) [reply data/i18n/hi/hi.toml:41]: Updated. Thanks.

**Comment by rolfedh** [comment]
Thank you for your thoughtful review, @Garima-Negi @Babapool! I hope now this PR might be ready for merge.

File: data/i18n/hi/hi.toml [hunk 6]
```diff
@@ -35,7 +35,10 @@ other = "आप इस कुबेरनेट्स संस्करण क
 other = " प्रलेखन अब सक्रिय रूप से मेंटेन नहीं है। वर्तमान में आप जो संस्करण देख रहे हैं वह एक स्टैटिक स्नैपशॉट है। अप-टू-डेट प्रलेखन के लिए, देखें "
 
 [deprecation_file_warning]
-other = "डेप्रिकॉटेड"
+other = "डेप्रिकेटड"
```
Comment (amolmote) [inline data/i18n/hi/hi.toml:38]:
```suggestion
other = "डेप्रिकेटेड"
```

File: data/i18n/hi/hi.toml [hunk 7]
```diff
@@ -35,7 +35,10 @@ other = "आप इस कुबेरनेट्स संस्करण क
...
+
+[dockershim_message]
+other = """1.24 संस्करण के बाद से Dockershim को कुबेरनेट्स परियोजना से हटा दिया गया हैं। अधिक जानकारी के लिए <a href="/dockershim">Dockershim रिमूवल एफएक्यू</a> पढ़ें।"""
```
Comment (amolmote) [inline data/i18n/hi/hi.toml:41]:
```suggestion
other = """1.24 संस्करण के बाद से डॉकरशिम को कुबेरनेट्स परियोजना से हटा दिया गया हैं। अधिक जानकारी के लिए <a href="/dockershim">डॉकरशिम रिमूवल एफएक्यू</a> पढ़ें।"""
```

File: data/i18n/hi/hi.toml [hunk 8]
```diff
@@ -88,6 +94,9 @@ other = ","
 [input_placeholder_email_address]
 other = "ईमेल ऐड्रेस"
 
+[javascript_required]
+other = "इस विषय को देखने के लिए जावास्क्रिप्ट [सक्रिय](https://www.enable-javascript.com/) होना चाहिए"
```
Comment (amolmote) [inline data/i18n/hi/hi.toml:98]:
```suggestion
other = "इस विषय को देखने के लिए जावास्क्रिप्ट [सक्रिय](https://www.enable-javascript.com/) होना चाहिए।"
```

**Review by amolmote — CHANGES_REQUESTED** [review]
these are the essential changes required, @anubha-v-ardhan have a look.

File: data/i18n/hi/hi.toml (hunk 6 above)
Comment (rolfedh) [reply data/i18n/hi/hi.toml:38]: Thanks for this suggestion @amolmote. [In a previous comment,](https://github.com/kubernetes/website/pull/36204#discussion_r954216012) @Garima-Negi suggested "डेप्रिकेटड". 

Garima, do you agree with this new suggestion?

File: data/i18n/hi/hi.toml (hunk 8 above)
Comment (rolfedh) [reply data/i18n/hi/hi.toml:98]: Updated. Thanks.

File: data/i18n/hi/hi.toml (hunk 6 above)
Comment (rolfedh) [reply data/i18n/hi/hi.toml]: Updated. Thanks, Amol.

**Comment by rolfedh** [comment]
I have verified that this PR merges smoothly and that the site builds and runs smoothly. I believe this PR is ready for merge.

**Review by anubha-v-ardhan — APPROVED** [review]
/lgtm
/assign
/approve

Thank you @rolfedh :)

**Pull request merged**